client/src/pages/proposalManagement/ProposalManagement.js

```diff
@@ -0,0 +1,158 @@
+        4 years.`,
+    },
+    
+    {
+      title: "Paint the Campanile Pink",
+      description: "The campanile should be pink!",
+    },
+  
+    {
+      title: "Make Big C Bigger!",
+      description: "MAKE BIG C BIG AGAIN!"
+    },
+  
+    {
+      title: "Give Oski A Makeover",
+      description: "Here's something we all know: Oski is creepy. We are asking for $1,000,000 to give him a makeover!",
+    },
+    
+    {
+      title: "Zero Waste 2020",
+      description: "Increase recycling by 200%",
+    },
+  
+    {
+      title: "Save The Dogs",
+      description: "Top dog saves dogs!"
+    },
+    
+    {
+      title: "BerkeleyMoves! Carsharing Pilot",
+      description: "The UC Berkeley Parking & Transportation Berkeley",
+    },
+  
+    {
+      title: "Plant 1,000,000 Trees",
+      description: "The goal is to plant 1,000,000 trees by the end of the year. ",
+    },
+  
+    {
+      title: "Trees for the Lorax",
+      description: "I speak for the trees",
+    },
+  
+    {
+      title: "ASUC Garden",
+      description: "We want to build a garden on Sproul",
+    },
+  
+    {
+      title: "Biofuels Technology R&D and Procurement",
+      description: "Repurpose waste cooking oil produced by Cal Dining facilities into biodiesel for use as a cleaner alternative fuel for campus vehicles."
+    },
+  
+    {
+      title: "Zero Waste Fellow",
+      description: "The creation of the Zero Waste Fellow position (similar to Carbon Neutrality or Engagement Fellows) will allow for institutionalized leadership that will continue to build upon and expand zero waste work."
+    },
+  
+    {
+      title: "Resilient Sustainibility Community Fellows",
+      description: "This project hopes to provide up to seven (7) post-baccalaureate fellowships for seniors."
+    }
+  ]
+
+function ProposalManagement() {
+    /* Contains all proposals. */
+    const proposalHTML = []
+    for (let i = 0; i < proposals.length; i++) {
+        proposalHTML.push(<Row changeTitle={(x) => {setProposalTitle(x)}} 
+                            changeDescription={(x) => {setProposalDescription(x)}}
+                            title={proposals[i].title} 
+                            description={proposals[i].description}
+                            displayX="true"
+                            x={xIcon}
+                            vote={proposals[i].voted ? proposals[i].voted : ""}
+                            />);
+    } 
+
+    const [proposalListDefault, setProposalListDefault] = useState(proposals);
+    const [proposalList, setProposalList] = useState(proposalHTML);
+
+    const [proposalTitle, setProposalTitle] = useState("Mapping for Environmental Justice");
+    const [proposalDescription, setProposalDescription] = useState("MEJ is an initiative to create interactive and publicly-accessible maps displaying environmental justice data for individual states.");  
+
+
+    
```

Comment:
just one line break should be enough; in general, try not to have too many unnecessary line breaks / whitespace